whitelabelco/sofo-foods
# Cp/feat/house brands copy

### Description 💻

Resolves: #158 & #159

### Implementation 💡

Updates Brands content according to ticket Reference #158 for copy 
Removes all copy from National Brands to display just logo as requested in #159 

#### Type of change


### How to test 🔍

- `npm i`
- `npm run dev`
- navigate to `http://localhost:3000/`
- click "Brands"
- confirm content matches what was described in the ticket

### Screenshots/Demos 📷

### Checklist ✅
